ansible_base/lib/dynamic_config/dynamic_settings.py

```diff
@@ -144,6 +144,10 @@
     REST_FRAMEWORK['DEFAULT_PAGINATION_CLASS'] = 'ansible_base.rest_pagination.DefaultPaginator'
 
 
+if 'ansible_base.jwt_consumer' in INSTALLED_APPS:
+    if 'ansible_base.rbac' not in INSTALLED_APPS:
+        INSTALLED_APPS.append('ansible_base.rbac')
```

Comment:
This would break galaxy_ng if they added the jwt_consumer to installed_apps.

https://github.com/ansible/galaxy_ng/blob/bc70fe3aecdeede74eb08f50fb3e5f3e8231d2f0/galaxy_ng/app/settings.py#L83

Right now I think they don't, and just use the auth class. But this does seem to undermine the point of having this switch. I would prefer to get rid of it. But non-blocking comment.

Reply:
galaxy_ng has issues for sure but that will be addressed laster.